### PR TITLE
[tensorflow][training] TF 2.13 SM Currency

### PR DESCRIPTION
**Pull request by xncqr**
*GitHub Issue #, if available:*

**Note**: 
- If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

- All PR's are checked weekly for staleness. This PR will be closed if not updated in 30 days.

### Description
* TF 2.13 SM build
* Update sagemaker-tensorflow-extensions from 1.18.0 to 1.19.0
* Skip benchmarks for TF 2.13 as they are not compatible. New benchmarks are being worked 

### Tests run
On Commit [3f134e5](https://github.com/aws/deep-learning-containers/pull/3204/commits/3f134e5cf06490599d512db58c48e7877dd2d469)
* Sanity Tests
* EC2 Tests
* ECS Tests
* EKS Tests
* SM Local Tests
* SM Remote Tests

On Commit [bcd4155](https://github.com/aws/deep-learning-containers/pull/3204/commits/bcd4155abceb48fb5edaa080fda912ac68d2a457)
* Benchmarks Tests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by tejaschumbalkar — DISMISSED** [review]

**Review by yl-to — DISMISSED** [review]
lgtm

File: tensorflow/training/docker/2.13/py3/Dockerfile.cpu
[the remark this answers is not in the document]
Comment (xncqr) [reply tensorflow/training/docker/2.13/py3/Dockerfile.cpu:1]: TY removing. is out of date.

File: tensorflow/training/docker/2.13/py3/cu118/Dockerfile.gpu
[the remark this answers is not in the document]
Comment (xncqr) [reply tensorflow/training/docker/2.13/py3/cu118/Dockerfile.gpu:1]: TY removing. is out of date.

**Review by tejaschumbalkar — APPROVED** [review]

**Review by 0x6b64 — APPROVED** [review]
LGTM! This PR has no changes w.r.t smddp/herring not being supported on Tensorflow.

**Review by yl-to — APPROVED** [review]

**Review by 0x6b64 — APPROVED** [review]
LGTM!

**Pull request merged**